frontend/src/Student/components/ShowCard.js

```diff
@@ -30,9 +75,23 @@ const ShowCard = (props) => (
       </Col>
     </Row>
     <hr />
-    <Row style={{ minHeight: '250px' }}>
-      <Col className='text-center align-self-center justify-content-center h-100'>
-        <Link to={`/submit?to=${props.show.id}`} className='d-block w-100 h-100'>
+    <Row style={{ minHeight: '250px' }} className='align-items-center'>
```

Comment:
My recommendation would be to have the flow of the submissions from right to left with the new submission being the rightmost item in the list

Reply:
R2L?

Wouldn't L2R seem more intuitive?

Reply:
It's more so where the new submission button ends up. I think it looks aesthetically more pleasing on the right hand side imo

Reply:
I think it's more that the new submission button should remain static for the user more than anything else

Reply:
I don't agree with you. In the case of needing to put the submission button on the new row, I think it looks better centered.  I feel that way about the last image in the gallery too.  Moving it with content doesn't make it hard to find.

Reply:
We can agree to disagree on this one. Rob can make the decision since its his PR